src/test/java/BasePage.java

```diff
@@ -0,0 +1,24 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {
```

Comment:
Please, move class BasePage in package "pages"

![image](https://user-images.githubusercontent.com/104761482/226826747-a48cc626-d716-491b-8f54-a6b2178f6bb6.png)